src/screens/proposal.tsx

```diff
@@ -21,9 +43,88 @@ export const ProposalScreen = () => {
     app.removeListener('wallet-selected-proposal-deposit', onDepositSubmit);
   };
 
+  const onDeposit = () => {
+    showModal('wallets-bottom-sheet', {
```

Comment:
if user have only one account these selector useless

Reply:
I made this modal so that it instantly disappears in this case and sends an event about the card selection